README.md

```diff
@@ -58,7 +58,7 @@ If you have 3D model animation problems, enable "Override software rendering lis
 
 ## Support
 
-If you need help your please use the multiwii or rcgroups forums or visit the IRC channel before raising issues in the issue trackers.
+If you need help please reach out on the [betaflightgroup slack channel](https://betaflightgroup.slack.com) before raising issues on github. 
```

Comment:
I think the link should rather be something like 'Betaflight slack ([registration](http://www.betaflight.tk/)', since users who don't have an account yet won't be able to access slack, and there is no link to the registration on the landing page.